### ISSUE TITLE
Small Visual Bug missing closing ")" under device  Data Query

### ISSUE THREAD

**Issue by ikorzha** [Cacti/cacti#854]
Latest cacti has missing closing ")" under device  Data Query. Been like that for a few releases
See picture attached:  (Look Under Status) for example at says active(1  but closing ")" is missing...

![missing](https://user-images.githubusercontent.com/11211789/28135756-512023b0-6715-11e7-8601-e81267648799.gif)

**Comment by cigamit** [comment]
Are you using php-snmp or net-snmp?

**Comment by ikorzha** [comment]
As per Cacti Technical Support Summary:
NET-SNMP version: 5.5

**Comment by cigamit** [comment]
What version of Cacti.  I can not reproduce this with 1.1.13.  There have been some changes in the snmp implementation (again) after the release of 1.x.  This was due to some realization that php-snmp is not sufficient for several tasks.  It needs an update.

**Comment by ikorzha** [comment]
I have actually just build a new system with Cacti 1.1.13 latest github on CentOS 7 and brand new Cacti install and I get identical results. I picked a sample cisco router, applied a cisco router template, clicked on "create graphs for this device"  look under "type"  do you see ethernetCsmacd(6    
it is missing closing ")"

Truth of the matter is if you can't replicate it I'm not too concerned it is a small visual bug.  I'm much more concerned with major errors I raised in "thold" branch that do not get addressed so far.
Bottom line if you can't replicate it please close it no need to clog ticketing system :) 
![image](https://user-images.githubusercontent.com/11211789/28465469-dd810396-6df6-11e7-8165-8765961bb708.gif)
....

**Comment by cigamit** [comment]
I have to figure out how it's possible on your installation.  For snmp walks, which is where we get this data, I have disabled getting the numeric data in the parenthesis from the snmp options.  Therefore, it should never appear at the end of the response.  Very odd.  It should not only be missing the end ')', but the beginning '(', and everything in the middle.  So, how it got there is beyond me.

**Comment by ikorzha** [comment]
With latest patch pre 1.1.16 the issue I have reported has been fixed. Thank you. I'm closing the case.
